openpype/hosts/maya/plugins/load/actions.py

```diff
@@ -114,13 +125,21 @@ def load(self, context, name=None, namespace=None, data=None):
         )
 
         with maintained_selection():
-            cmds.file(self.fname,
-                      i=True,
-                      preserveReferences=True,
-                      namespace=namespace,
-                      returnNewNodes=True,
-                      groupReference=True,
-                      groupName="{}:{}".format(namespace, name))
+            nodes = cmds.file(self.fname,
+                              i=True,
+                              preserveReferences=True,
+                              namespace=namespace,
+                              returnNewNodes=True,
+                              groupReference=True,
+                              groupName="{}:{}".format(namespace, name))
+
+            if clean_import:
+                shapes = cmds.ls(nodes, shapes=True, long=True)
+                for shape in shapes:
+                    meshes = cmds.ls('{}.cbId'.format(shape))
```

Comment:
This should be `cmds.attributeQuery("cbId", node=shape, exists=True)` to be more explicit.
I'd also disregard the filtering to shapes and just check on all nodes because transforms of shapes also get a cbId.
```python
remove_attributes = ["cbId"]
for node in nodes:
    for attr in remove_attributes:
        if cmds.attributeQuery(attr, node=node, exists=True):
            cmds.deleteAttr("{}.{}".format(attr))